lib/taiko.js

```diff
@@ -2042,6 +2043,70 @@ module.exports.link = (attrValuePairs, ...args) => {
   };
 };
 
+/**
+ *
+ * Function to select elements which contains specific Text
+ *
+ * @example
+ * await click(containsText('Get Started'))
+ * await containsText('Get Started').exist()
+ *
+ * @param {string} text - The link text.
+ * @param args - Proximity selectors
+ * @returns {ElementWrapper}
+ */
+module.exports.containsText = (text, ...args) => {
+  validate();
+  assertType(text);
+  let nbspChar = String.fromCharCode(160);
+  let textToTranslate = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' + nbspChar;
+  let translateTo = 'abcdefghijklmnopqrstuvwxyz' + ' ';
+  let xpathText = `translate(normalize-space(${xpath(
+      text,
+  )}), "${textToTranslate}", "${translateTo}")`;
+  const get = async () => await handleRelativeSearch(await $$xpath(
+      `//text()[contains(translate(normalize-space(string()), "${textToTranslate}", "${translateTo}"), ${xpathText})]`,
+  ),args);
```

Comment:
Can't we use the exists fn - https://github.com/getgauge/taiko/pull/866/files#diff-fa054c0cd359737bbdc57062a6fee95aR122

Reply:
Can the logic be moved to `elementSearch.js`? @nehashri Thoughts?